dev-tools/validation/test_check_dev_demo_summary.py

```
@@ -0,0 +1,391 @@
+    spec = importlib.util.spec_from_file_location("dev_demo_summary_validator", SCRIPT)
+    if spec is None or spec.loader is None:
+        raise AssertionError("could not load dev-demo summary validator")
+    module = importlib.util.module_from_spec(spec)
+    sys.modules[spec.name] = module
+    spec.loader.exec_module(module)
+    return module
+
+
+class DevDemoSummaryValidatorTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.16.0)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)